File: HlloGeegBrains/app/src/main/java/com/example/hllogeegbrains/MainActivity.java
package com.example.hllogeegbrains;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity implements View.OnClickListener {

    private Button button;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        button = findViewById(R.id.button);
        button.setOnClickListener(this);
    }

    public void onMyClick(View view) {
        TextView textView = findViewById(R.id.textView);
        textView.setVisibility(View.VISIBLE);
    }

    @Override
    public void onClick(View view) {
        if (view.getId() == R.id.button) {
            Intent intent = new Intent(this, SideActivity.class);
            startActivity(intent);
        }
        if (view.getId() == R.id.toggleButton) {
            TextView textView = findViewById(R.id.textView2);
            if (textView.getVisibility() == View.INVISIBLE) {
                textView.setVisibility(View.VISIBLE);
            } else {
                textView.setVisibility(View.INVISIBLE);
            }
        }
    }
}

File: Kotlin/Weather/app/src/main/java/com/example/weather/MainActivity.kt
package com.example.weather

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.google.android.material.button.MaterialButton

class MainActivity : AppCompatActivity() {

    private lateinit var text: TextView
    private lateinit var button: MaterialButton

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val note = Note("title", "name")
        val note1 = note.copy()
        val textToView = note1.title + " " + note1.note

        text = findViewById(R.id.text_view)
        button = findViewById(R.id.button)
        val cobbledString = nextCycle(cycle(), textToView)
        button.setOnClickListener { text.text = cobbledString }
    }

    private fun cycle(): List<String> {
        val daysOfWeek =
            listOf(resources.getString(R.string.app_name), "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday")
        val daysWithIndex = mutableListOf<String>()
        daysOfWeek.forEachIndexed { index, s ->
            val temp = "$s ${index + 1}"
            daysWithIndex.add(temp)
        }
        return daysWithIndex
    }

    private fun nextCycle(list: List<String>, textToView: String): String {
        val s = StringBuilder()
        for (i in list.indices) {
            s.append(list[i])
                .append(" ")
                .append(textToView)
                .append(" ")
                .append(i + 1)
                .append("\n")
        }
        return s.toString()
    }
}